src/components/lead/CdrLead.vue

```diff
@@ -1,24 +1,40 @@
 <script setup lang="ts">
-import { useCssModule } from 'vue';
+import { useCssModule, computed } from 'vue';
+import mapClasses from '../../utils/mapClasses';
 
 /** Container used for assembling leads */
 defineOptions({
   name: 'CdrLead'
 });
 
-const style = useCssModule();
+interface leadProps {
+  /** 
+   * Sets the orientation of the background surface
+   * @demoSelectMultiple false
+   * @values 'top', 'bottom'
+   */
+  surfaceOrientation: 'top' | 'bottom'
```

Comment:
surfaceOrientation seems a bit of a mouthful would surface:top /bottom do the same?